src/psim/sensors/gyroscope.cpp

```diff
@@ -33,15 +33,19 @@ namespace psim {
 void Gyroscope::step() {
   this->Super::step();
 
-  // TODO : Implement gyroscope bias drift
+  auto       &bias = sensors_satellite_gyroscope_w_bias.get();
+  auto const &bias_sigma = sensors_satellite_gyroscope_w_bias_sigma.get();
+  auto const &dt = truth_dt_s->get();
+
+  bias = bias + dt * lin::multiply(bias_sigma, lin::gaussians<Vector3>(_randoms));
 }
 
 Vector3 Gyroscope::sensors_satellite_gyroscope_w() const {
   auto const &truth_w = truth_satellite_attitude_w->get();
-  auto const &bias = sensors_satellite_gyroscope_bias.get();
+  auto const &bias = sensors_satellite_gyroscope_w_bias.get();
+  auto const &sigma = sensors_satellite_gyroscope_w_sigma.get();
 
-  // TODO : Implement a white noise model
-  return truth_w + bias;
+  return truth_w + bias + lin::multiply(sigma, lin::gaussians<Vector3>(_randoms));
```

Comment:
I imagine this kind of modeling (bias + truth) will be extremely pervasive across many sensor models, should we build some kind of library that will automatically figure out the right state fields to use for offsetting truth?

Reply:
Believe it or not, this is the only sensor on the spacecraft that has a bias worth modelling. That being said, the generic white noise model is a bit more pervasive in the code base (pretty much every other sensor).

If we try to generalize that model we may be able to. That being said though we'd lose all the sensor specific comments in the YAML file and things wouldn't be documented as well. For that reason I'm hesitant to generalize the sensor model to a single interface with more arguments.

Reply:
Let's just leave things as is.